src/styles/_normalize.scss

```
@@ -0,0 +1,5 @@
+body {
+  background-color: rgba(206, 202, 209, 0.91);
```

```suggestion
.body {
  background-color: rgba(206, 202, 209, 0.91);
```

we don't use tags for styling